src/graph/backend/dnnl/kernels/sdp_decomp_config.cpp

```diff
@@ -65,10 +71,10 @@ bool sdp_decomp_config_t::initial_check(const std::shared_ptr<subgraph_t> &sg,
 #define RATIO 2
     // Initialize nthr with current threads num
     nthr = dnnl_get_current_num_threads();
-    return batch_size * num_head_q > RATIO * nthr;
-#else
-    return true;
+    VCHECK_SDP_DECOMP(batch_size * num_head_q > RATIO * nthr, false,
+            "Batch size * num_head_q should be larger than %d * nthr", RATIO);
```

Comment:
This is also a kernel limitation. I'm not sure if the message clarifies that.

Reply:
added more info.